lib/devices.go

```diff
@@ -111,7 +131,7 @@ func getDevices() (time.Time, []Device) {
 	})
 	if err != nil {
 		log.Printf("Issue accessing USB Devices: %v", err)
```

Comment:
You're now generating generic error messages to the log instead of adding this specific one there.